server/blob/blob-cassandra/src/main/java/org/apache/james/blob/cassandra/cache/CassandraBlobStoreCache.java

```diff
@@ -86,7 +97,7 @@ public Mono<byte[]> read(BlobId blobId) {
                 selectStatement.bind()
                     .setString(ID, blobId.asString())
                     .setConsistencyLevel(ONE)
-                    .setReadTimeoutMillis(readTimeOutFromDataBase))
+                    .setTimeout(Duration.ofDays(readTimeOutFromDataBase)))
```

Comment:
Are we sure?

Reply:
My fault

Reply:
> My fault

Why is this a fault?

In my work, I even set a dedicated profile for this timeout. But turnout we have this `.setTimeout`.
Seem like Cassandra driver 4 does not have a dedicated timeout for read/write, it just has `REQUEST_TIMEOUT("basic.request.timeout")`.

Reply:
Old: Millisecond 
My code: Day

It should be 

```suggestion
.setTimeout(Duration.ofMillis(readTimeOutFromDataBase)))
```
I got an issue with Cassandra query timeouts, I modified it for debug and forget to revert

Reply:
Ah ok I got it.

Reply:
To be set via a profile!